l10n/de_DE.ts

```diff
@@ -1,22 +1,22 @@
 export default {
   search: {
-    placeholder: 'Geben Sie ein Schlüsselwort ein...',
+    placeholder: 'Suchbegriff eingeben...',
   },
   sort: {
     sortAsc: 'Spalte aufsteigend sortieren',
     sortDesc: 'Spalte absteigend sortieren',
   },
   pagination: {
-    previous: 'Bisherige',
+    previous: 'Vorherige',
     next: 'Nächste',
     navigate: (page, pages) => `Seite ${page} von ${pages}`,
     page: (page) => `Seite ${page}`,
-    showing: 'Anzeigen',
+    showing: 'Datensätze',
```

Comment:
You can't use the "showing x of y" in German, so I had to get a little creative. In the perfect world, the whole "sentence" would be a translatable.